Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -599,10 +599,10 @@ def send_report(self):
                       format(self.k_folds))]
         elif self.resampling == self.LeaveOneOut:
             items = [("Sampling type", "Leave one out")]
-        elif self.resampling == self.Bootstrap:
-            items = [("Sampling type",
+        elif self.resampling == self.ShuffleSplit:
+             items = [("Sampling type",
                       "{} random samples with {} % data ".format(
```

Comment:
This line (and the subsequent lines) should be indented one space further. The string is _inside_ the tuple.